lib/wise_homex/models/property.ex

```diff
@@ -7,6 +7,7 @@ defmodule WiseHomex.Property do
     belongs_to :admin, WiseHomex.Account
     belongs_to :external_info, WiseHomex.ExternalInfo
     has_many :addresses, WiseHomex.Address
+    has_many :statements, WiseHomex.Statement
```

Comment:
Just a note: A property also has many fiscal_years. We don't need to include that here, though.

Reply:
Let's do that when we get to fiscal years